File: solutions/q1480/q1480.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        runSum = [nums[0]] 
	    
        for i in range(1, len(nums)):
		    runSum.append(runSum[i-1] +nums[i]) 
	    
        return runSum

File: solutions/q1480/q1480.cs
public class Solution {
    public int[] RunningSum(int[] nums) {
        int runningSum = 0;
        for(int i = 0; i < nums.Length; i++)
        {
            runningSum += nums[i];
            nums[i] = runningSum;
        }
        return nums;
    }
}
